docker-compose.migrate.yaml

```diff
@@ -1,5 +1,3 @@
-version: '3.8'
```

Comment:
Does it work without this line?

Reply:
@kbabigumira version is obsolete now and it works fine without it